README.md

```diff
@@ -176,32 +176,36 @@ you just need to repeat the steps the desired JavaScript filename.
 
 #### x86 application image only
 If you want to build the x86 and ARC images separately, or if you just want to
-build another .js sample that uses the same sub-modules, then you can  build x86
-only with
+build another .js sample that uses the same sub-modules, then you can build only
+the x86 image with:
 
 ```bash
-$ make JS=samples/xxx.js
-```
+$ make JS=path/to/filename.js
 
-```bash
 $ make dfu
 ```
 
 #### ARC support image only
-To build and flash the ARC support image only
+To build and flash only the ARC support image:
 
 ```bash
-$ make JS=samples/xxx.js arc
-```
+$ make JS=path/to/filename.js arc
 
-```bash
 $ make dfu-arc
 ```
 
 If you run a different .js sample that uses different sub-modules, for example,
 from AIO.js to I2C.js, then you'll need to repeat the last two steps to flash
 both images.
 
+If you forget to build and update the ARC image, or somehow you build the x86
+and ARC images, you'll see an error on your serial console (see below) similar
```

Comment:
I'd replace with this:

If you forget to update the ARC image when your module requirements have changed, you'll see an error...

(I don't get what the "or somehow you build..." part was supposed to mean.)